quicktest.go

```diff
@@ -0,0 +1,106 @@
+// Licensed under the MIT license, see LICENCE file for details.
+
+package quicktest
+
+import (
+	"fmt"
+	"testing"
+)
+
+// New returns a new quick test checker, that is used to call Assert and Check.
+// The checker is instantiated passing the *testing.T value for the current
+// test or subtest. For instance:
+//
+//     func TestFoo(t *testing.T) {
+//         t.Run("A=42", func(t *testing.T) {
+//             c := qt.New(t)
+//             c.Assert(a, qt.Equals, 42)
+//         })
+//     }
+//
+// The library provides the Equals, CmpEquals, DeepEquals, ErrorMatches,
+// PanicMatches, IsNil and Not checkers. More can be added by implementing the
+// Checker interface.
+func New(t T) *C {
+	return &C{
+		t: t,
+	}
+}
+
+// C is a quick test checker used to call Assert and Check.
+type C struct {
+	t T
+}
+
+// Check runs the given check and continues execution in case of failure.
```

Comment:
// Please document how the additional args parameter is used.

Reply:
Done.